google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -115,13 +123,21 @@ appengine.stop.modules.version.success.dialog.title=Application Stopped
 appengine.stop.modules.version.canceled.message=Undeploy Canceled
 appengine.deployment.error.during.staging=Deployment failed due to staging failing with exit code: {0}.
 appengine.deployment.exception.during.staging=Deployment failed due to an exception while staging the project.
+appengine.deployment.error.staging.yaml=The specified YAML configuration file does not exist or is not a valid file.
+appengine.deployment.error.staging.dockerfile=The specified Dockerfile configuration file does not exist or is not a valid file.
+appengine.deployment.error.staging.gotosettings=Set a valid file in Module Settings or use another one.
```

Comment:
This is appended to the immediately above messages, when an invalid file is selected from the Flex deployment UI.
I can't add this message to the other ones because they're used elsewhere.